docs/examples/use_cases/paddle/resnet50/main.py

```diff
@@ -24,97 +24,62 @@
-        self.decode = ops.ImageDecoderRandomCrop(
-            device='mixed',
-            output_type=types.RGB,
-            device_memory_padding=device_memory_padding,
-            host_memory_padding=host_memory_padding,
-            random_aspect_ratio=[0.8, 1.25],
-            random_area=[0.1, 1.0],
-            num_attempts=100)
-        self.res = ops.Resize(device='gpu',
-                              resize_x=crop,
-                              resize_y=crop,
-                              interp_type=types.INTERP_TRIANGULAR)
-        self.cmnp = ops.CropMirrorNormalize(
-            device="gpu",
-            dtype=types.FLOAT,
-            output_layout=types.NCHW,
-            crop=(crop, crop),
-            mean=[0.485 * 255, 0.456 * 255, 0.406 * 255],
-            std=[0.229 * 255, 0.224 * 255, 0.225 * 255])
-        self.coin = ops.CoinFlip(probability=0.5)
-        self.to_int64 = ops.Cast(dtype=types.INT64, device="gpu")
-
-    def define_graph(self):
-        rng = self.coin()
-        jpegs, labels = self.input(name="Reader")
-        images = self.decode(jpegs)
-        images = self.res(images)
-        output = self.cmnp(images.gpu(), mirror=rng)
-        return [output, self.to_int64(labels.gpu())]
-
-    def __len__(self):
-        return self.epoch_size("Reader")
-
-
-class HybridValPipe(Pipeline):
-    def __init__(self):
-        super(HybridValPipe, self).__init__(FLAGS.batch_size,
-                                            FLAGS.num_threads,
-                                            FLAGS.device_id,
-                                            seed=42 + FLAGS.device_id)
-        data_dir = os.path.join(FLAGS.data, 'val')
-        self.input = ops.FileReader(file_root=data_dir,
-                                    shard_id=0,  # XXX eval only on rank 0
-                                    num_shards=1,
-                                    random_shuffle=False,
-                                    pad_last_batch=True)
-        self.decode = ops.ImageDecoder(device="mixed", output_type=types.RGB)
-        self.res = ops.Resize(device="gpu",
-                              resize_shorter=256,
-                              interp_type=types.INTERP_TRIANGULAR)
-        self.cmnp = ops.CropMirrorNormalize(
-            device="gpu",
-            dtype=types.FLOAT,
-            output_layout=types.NCHW,
-            crop=(224, 224),
-            mean=[0.485 * 255, 0.456 * 255, 0.406 * 255],
-            std=[0.229 * 255, 0.224 * 255, 0.225 * 255])
-        self.to_int64 = ops.Cast(dtype=types.INT64, device="gpu")
-
-    def define_graph(self):
-        jpegs, labels = self.input(name="Reader")
-        images = self.decode(jpegs)
-        images = self.res(images)
-        output = self.cmnp(images)
-        return [output, self.to_int64(labels.gpu())]
-
-    def __len__(self):
-        return self.epoch_size("Reader")
-
+def get_dali_pipeline(batch_size, num_threads, device_id, data_dir, crop, size,
```

Comment:
```suggestion
def create_dali_pipeline(batch_size, num_threads, device_id, data_dir, crop, size,
```
or
```suggestion
def build_dali_pipeline(batch_size, num_threads, device_id, data_dir, crop, size,
```
(in the latter case we can call `pipe.build()` inside)

Reply:
Done